app/experimenter/experiments/tests/api/v5/test_serializers.py

```diff
@@ -126,7 +143,69 @@ def test_saves_existing_experiment_with_changelog(self):
         self.assertEqual(experiment.changes.count(), 2)
 
 
-class TestUpdateNimbusExperimentOverviewSerializer(TestCase):
+class TestNimbusBranchSerializer(TestCase):
+    def test_branch(self):
```

Comment:
If this is testing that a valid branch passes validation I might just say something about that in the test name